libs/designsystem/src/lib/components/chart/chart.component.spec.ts

```diff
@@ -5,6 +5,31 @@ import { MockProvider } from 'ng-mocks';
 import { ChartJSService } from './chart-js/chart-js.service';
 import { ChartComponent } from './chart.component';
 
+describe('ChartComponent in component with out height & width', () => {
+  let spectator: Spectator<ChartComponent>;
+  let component: ChartComponent;
+  const createHost = createHostFactory({
+    component: ChartComponent,
+    declarations: [ChartComponent],
+    componentProviders: [MockProvider(ChartJSService)],
+  });
+
+  beforeEach(() => {
+    spectator = createHost(
+      '<div style="height: 0px; width: 0px;"><kirby-chart></kirby-chart></div>'
+    );
+    component = spectator.component;
+  });
+
+  it('should not render if no height and width', () => {
+    const redrawChartSpy = spyOn<any>(component, 'updateDataLabels');
+
+    spectator.setInput('dataLabels', ['1', '2']);
+
+    expect(redrawChartSpy).toHaveBeenCalledTimes(0);
+  });
```

Comment:
Looking at this with fresh eyes, i think we could restructure it a bit to follow the [given-when-then formula](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test#use-the-given-when-then-formula) a bit more: 

``` typescript
describe('when a chartJS related input property is changed', () => {
   let updateFnSpy;

   beforeEach(() => 
      updateFnSpy = spyOn<any>(component, 'updateDataLabels);
      spectator.setInput('dataLabels', ['1', '2']);
   );
   
   it('should not call the corresponding update function', () =>{
      expect(updateFnSpy).toHaveBeenCalledTimes(0);
   });
`